### PR TITLE
LTG92 Dragino decoder

### PR DESCRIPTION
**Pull request by Jaime-Trinidad**
#### Summary

Decoder now is updated as Dragino information.

#### Changes

- latitude and longitude decoder

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/dragino/lgt92.js [hunk 1]
```diff
@@ -2,26 +2,13 @@ function decodeUplink(input) {
...
-{
-  data.longitude=(bytes[4]<<24 | bytes[5]<<16 | bytes[6]<<8 | bytes[7])/1000000;//gps longitude,units: °
-}
-else
-{
- data.longitude=0;//gps longitude,units: °
-}  
+
+data.latitude=(bytes[0]<<24 | bytes[1]<<16 | bytes[2]<<8 | bytes[3])/1000000;//gps latitude,units: °
+data.longitude=(bytes[4]<<24 | bytes[5]<<16 | bytes[6]<<8 | bytes[7])/1000000;//gps longitude,units: °
```
Comment (johanstokking) [inline vendor/dragino/lgt92.js:11]: This doesn't seem to account for negative values

**Comment by Jaime-Trinidad** [comment]
@johanstokking I fix it to consider negative values, it will send 0.

**Comment by johanstokking** [comment]
But these values _can_ be negative

**Comment by Jaime-Trinidad** [comment]
@johanstokking last version still get values if msb is 0, this can solve #331 what do you think?

**Comment by johanstokking** [comment]
> @johanstokking last version still get values if msb is 0, this can solve #331 what do you think?

Which last version?

I agree we shouldn't use MSB as signaling whether the value is set.

Still we don't account for negative values here.

**Pull request closed without merging**